### ISSUE TITLE
Feature Request: Adding a count down for autoClose

### ISSUE THREAD

**Issue by SinghChandrabhan** [ngneat/hot-toast#73]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[x] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

I guess this is an end to my ever lasting search for an Angular toast (the under budget package size is a cherry on the cake ).
I still want to see onle more feature inbuild(greedy). So,   when you set `autoClose` property the toast would close automatically(mouse over will delay the closing). 

## Expected behavior

I am wondering if it is possible to put a count down showing how long the toast will be displayed(or even a decreasing progress bar). For example , when you delete a message in the Telegram app, it shows a similar couter (image attached)

![image](https://user-images.githubusercontent.com/20701328/124491212-4d423080-ddb3-11eb-8af4-d4942c42f919.png)

## Minimal reproduction of the problem with instructions
1) Use ngneat hot toast to display a toast message
2) Set `autoClose` property to true in `toast.show` function

## What is the motivation / use case for changing the behavior?
A count down shown on the toast would increas the user experience, User would have an idea when the toast is going to be closed.

## Environment

<pre><code>
Angular version: 12.0

Browser:
- [x] Chrome (desktop) version XX
- [x] Chrome (Android) version XX
- [x] Chrome (iOS) version XX
- [x] Firefox version XX
- [x] Safari (desktop) version XX
- [x] Safari (iOS) version XX
- [x] IE version XX
- [x] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

**Comment by shhdharmen** [comment]
I don't think this feature is generally needed. It's a very specific need. You can create the timer with needed UI in [template](https://ngneat.github.io/hot-toast/#template) or [component](https://ngneat.github.io/hot-toast/#component) approach.

**Comment by shhdharmen** [comment]
Let's keep it open and see the demand for this feature.

**Comment by daboudii** [comment]
It would be a nice feature, other solution could also be a progress bar under the toastr

![Screenshot from 2021-10-19 09-53-02](https://user-images.githubusercontent.com/52279206/137858747-234037d2-4195-4c47-ad6c-033058ed548e.png)

**Comment by shhdharmen** [comment]
@daboudii for progress bar, you create component and use it with toast.
